core/src/main/resources/jenkins/install/platform-plugins.json

```diff
@@ -4,6 +4,7 @@
         "plugins": [
             { "name": "dashboard-view" },
             { "name": "cloudbees-folder", "suggested": true },
+            { "name": "configuration-as-code", "suggested": true },
```

Comment:
```suggestion
            { "name": "configuration-as-code" },
```

Reply:
we can probably just hold a bit and see how milestones go? unless @casz wants this in now?

Reply:
Shall we make this change to add it now and then add suggested as a follow-up?

Reply:
please do @timja